### PR TITLE
fix: update header style

### PR DESCRIPTION
**Pull request by petey**
Context:
========
Header was still using our old, very colorful style

Objective:
----------
Make header match the cd.screwdriver.cd style

screen shot 2017-07-10 at 12 02 59 pm

### CODE REVIEW

File: themes/sd/css/base.css [hunk 1]
```diff
@@ -144,11 +144,12 @@ footer .container > div:last-child {
  * Top Navigation
  */
 .navbar {
-    background: url('/img/main_bkgrd_home.png') no-repeat;
-    background-size: cover;
+    background: #333;
+    /*background: url('/img/main_bkgrd_home.png') no-repeat;*/
```
Comment (tkyi) [inline themes/sd/css/base.css]: Should we delete this code?

**Comment by tkyi** [comment]
Ooh. Should the Search bar also match? Or nah?

File: themes/sd/css/base.css [hunk 2]
```diff
@@ -144,11 +144,10 @@ footer .container > div:last-child {
  * Top Navigation
  */
 .navbar {
-    background: url('/img/main_bkgrd_home.png') no-repeat;
-    background-size: cover;
+    background: #333;
 }
 .navbar-brand {
-    background: transparent url('/svg/Screwdriver_Logo_White.svg') no-repeat;
```
Comment (ian-fox) [inline themes/sd/css/base.css:151]: Should we delete this file? It doesn't look like we're using Screwdriver_Logo_White anywhere if this is gone.

**Comment by petey** [comment]
@tkyi There isn't a search bar in the header, only as a separate popup. I don't intend to change that.

**Review by tkyi — APPROVED** [review]

**Pull request merged**